client/src/app/pages/archetypes/archetypes-page.tsx

```diff
@@ -367,26 +380,44 @@ const Archetypes: React.FC = () => {
                         <Td isActionCell>
                           <ActionsColumn
                             items={[
-                              {
-                                title: t("actions.duplicate"),
-                                onClick: () =>
-                                  setArchetypeToDuplicate(archetype),
-                              },
-                              {
-                                title: t("actions.assess"),
-                                onClick: () =>
-                                  assessSelectedArchetype(archetype),
-                              },
-                              {
-                                title: t("actions.review"),
-                                onClick: () =>
-                                  reviewSelectedArchetype(archetype),
-                              },
-                              {
-                                title: t("actions.edit"),
-                                onClick: () => setArchetypeToEdit(archetype),
-                              },
-                              ...(archetype?.assessments?.length
+                              ...(archetypeWriteAccess
+                                ? [
+                                    {
+                                      title: t("actions.duplicate"),
+                                      onClick: () =>
+                                        setArchetypeToDuplicate(archetype),
+                                    },
+                                  ]
+                                : []),
```

Comment:
The alternate form would be to drop undefieneds and then `.filter(Boolean)` at the end...

```js
         [
           archetypeWriteAccess && {
              title: ...
             ...
           },
           assessmentWriteAccess && { ... },
           ...
         ].filter(Boolean)
```